frontend/src/screens/learner/card.tsx

```diff
@@ -126,150 +118,325 @@ const CompleteFlag: React.FC<StudyCardProps> = ({ study }) => {
     if (!study.completedAt) return null
 
     return (
-        <Box gap
-            align="center"
-            padding="default"
-            css={{
-                backgroundColor: colors.green,
-                position: 'absolute',
-                borderBottomLeftRadius: 20,
-                borderTopLeftRadius: 20,
-                right: 0,
-                top: 16,
-            }}
-        >
-            <Icon icon="checkCircle" color='white' />
-            <span>Complete</span>
-        </Box>
+        <Flex justify='center' align='center' pos='absolute' top='-.5rem' right='-.5rem' style={{ zIndex: 2 }}>
+            <div style={{ width: '2rem', 
+                height: '2rem', 
+                backgroundColor: colors.purple, 
+                borderRadius: '50%',
+                boxShadow: '-2px 2px 10px rgba(0, 0, 0, 0.20)',
+            }}></div>
+            <Icon icon='thickCheck' color={colors.white} 
+                css={{
+                    position: 'absolute',
+                    fontSize: '1rem',
+                }}/>
+        </Flex>
+    )
+}
+
+const MultiSessionBar: FC<StudyCardProps> = ({ study }) => {
+    if (!study.stages || study.stages.length < 2) return null
+    return (
+        <Flex justify='center' align='flex-start' direction='column' w='100%'>
```

Comment:
any reason why you dont use a `<Stack>` here?

Reply:
Stack can be used, I'll modify it